pkg/plugins/envvars/envvars.go

```diff
@@ -91,10 +92,10 @@ func (s *Service) GetConfigMap(ctx context.Context, _ string, _ *oauth.ExternalS
 			for feat := range enabledFeatures {
 				features = append(features, feat)
 			}
+			sort.Strings(features)
```

Comment:
can you add tests for this function?

Also, unrelated to this PR but I just noticed that this `GetConfigMap` is within the `envvars` package but it has "nothing" to do with envvars (only that it receives the same input). This could be confusing in the future, maybe it's worth to move it to a new Service? (or an existing one but one that is more relevant like `plugins/config`)

Reply:
Yep will add some tests!

I plan to have a follow up PR to move this eventually. Is just convenient for now 😄